cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1565,9 +1565,8 @@ function Install_Containerd {
   $tmp_dir = 'C:\containerd_tmp'
   New-Item $tmp_dir -ItemType 'directory' -Force | Out-Null
 
-  # TODO(ibrahimab) Change this to a gcs bucket with CI maintained and accessible by community.
```

Comment:
FWIW we host binaries for kind publicly as-is, though it is also unofficial https://github.com/kind-ci/containerd-nightlies

containerd also has amd64 binaries on their releases page

Reply:
IMO this TODO still applies, to anything in this repo.

Reply:
Yeah, I actually added the TODO when I switched it from the temp gcs bucket used earlier. The problem of keeping it only tied to the binaries available through release links, is that sometimes with certain cherry-picks needed we've to wait till it gets picked by a following release (this is the case with the csi driver issue mentioned above - that's causing failures in the gce grid). So those binaries are built from the main github containerd repo with just the cherry-pick applied and pushed similar to all other components to gke-release.

We can switch back later after releases becomes more stable, as I agree that it'll be cleaner.

Do you suggest keeping the TODO ?

Reply:
I think this is still a TODO: we can host nightly binaries on in-project resources. I also don't think we need to be running from a fork vs the latest changes in a given branch including the cherry pick there, at least on linux this works fine and we're regularly running CI not at a release commit.

Reply:
Sgtm! Kept the TODO.